test/scip/testdata/enum.snapshot.rb

```diff
@@ -10,23 +9,45 @@ class X < T::Enum
 #             ^^^^ reference [..] T#Enum#
    enums do
      A = new("A")
-#    ^ definition local 2~#119448696
-#    ^ definition [..] X#A#
-#    ^ reference [..] X#A#
-#    ^ reference [..] X#A.
+#    ^ definition [..] X#A.
```

Comment:
The main change is that this correctly shows up as a definition rather than as a reference, and that spurious entities are no longer present here.